Add Audio input/output tokens as special columns, filling in for Gemini 2.5 Flash

niharm opened the pull request:
This PR adds  "Audio input cost" and "audio output cost" as special columns. 

As noted in https://github.com/sst/models.dev/pull/223, Gemini 2.5 Flash (and other models) have special input costs for Audio. This price isn't currently captured in Models.dev.

<img width="2266" height="668" alt="image" src="https://github.com/user-attachments/assets/a3c8ee07-ea9a-4f97-b4ba-882731cb1652" />

This PR proposes adds data for Gemini 2.5 Flash related to special input/output audio tokens, as well as a new model for "Gemini Live 2.5 Flash Preview Native Audio".

This approach is also taken by others such as LiteLLM -- see `input_cost_per_audio_token` in [this file](https://github.com/BerriAI/litellm/blob/main/model_prices_and_context_window.json)

niharm committed:
Add audio input as a column

niharm commented on providers/google/models/gemini-2.5-flash.toml:
```diff
@@ -12,6 +12,7 @@ open_weights = false
 input = 0.30
 output = 2.50
 cache_read = 0.075
+input_audio_token = 1.00
```
Could also call this `input_audio`

fwang commented:
Thanks @niharm!

Yeah let's call it `input_audio`. Can you also do a quick grep and see whereelse to document this. I think readme on top of my head.

niharm committed:
PR Feedback

niharm committed:
Add output_audio and gemini 2.5 flash lite preview

niharm commented:
Thanks @fwang!

I edited the readme and verified there are no other spots. 

```
grep -r cache_write . --exclude-dir=providers -l
./README.md
./packages/core/src/schema.ts
./packages/web/src/render.tsx
```

I also added output tokens in the same format (again, similar to [LiteLLM](https://github.com/BerriAI/litellm/blob/81282d14b5196eccdc59c0c993ea16b19d08bcff/model_prices_and_context_window.json#L8963))

I added a sample model that has special input/output audio (and video) pricing: https://ai.google.dev/gemini-api/docs/pricing#gemini-2.5-flash-native-audio

Updated this PR.

niharm commented on providers/google/models/gemini-live-2.5-flash-preview-native-audio.toml:
```diff
@@ -0,0 +1,24 @@
+name = "Gemini Live 2.5 Flash Preview Native Audio"
```
From https://cloud.google.com/vertex-ai/generative-ai/docs/models/gemini/2-5-flash#live-api-native-audio and https://ai.google.dev/gemini-api/docs/pricing#gemini-2.5-flash-native-audio

fwang commented:
Thanks @niharm!
